docs/how-tos/use_the_api.rst

```diff
@@ -0,0 +1,86 @@
+How To Use the REST API
+#######################
+
+.. How-tos should have a short introduction sentence that captures the user's goal and introduces the steps.
+
+This how-to will help get setup to be able to make aunthenticated requests to
```

Comment:
[inform] You may want to review https://edx-enterprise-api.readthedocs.io/en/latest/authentication.html to see if there is any details from there you'd like to add. Note that @pomegranited asked elsewhere about token expiration, and this docs has something on that.

Reply:
Cross linked for now.